resolve/vector/VectorHandlerCpu.cpp

```diff
@@ -154,9 +183,27 @@ namespace ReSolve {
    * @pre   _k_ > 0, _size_ > 0, _size_ = x->getSize()
    *
    */
-  void VectorHandlerCpu::massAxpy(index_type /* size */, vector::Vector* /* alpha */, index_type /* k */, vector::Vector* /* x */, vector::Vector* /* y */)
+  void VectorHandlerCpu::massAxpy(index_type size, 
+                                  vector::Vector* alpha, 
+                                  index_type k, 
+                                  vector::Vector* x, 
+                                  vector::Vector* y)
   {
-    out::error() << "Not implemented (yet)" << std::endl;
+  
+    real_type* alpha_data = alpha->getData(memory::HOST);
+    real_type* y_data = y->getData(memory::HOST);
+    real_type* x_data = x->getData(memory::HOST);
+    index_type i, j;
+    real_type sum;
+    
+    for (i = 0; i < size; ++i) {
+      sum = 0.0;
+      for (j = 0; j < k; ++j) {
+     // if(i == 2)  printf("size x %d size alpha %d multiplying x[%d] = %16.16e by alpha[%d] = %16.16e \n",size *k, k, j * size + i, x_data[j * size + i], j, alpha_data[j] );
+        sum += x_data[j * size + i] * alpha_data[j];
```

Comment:
Just get rid of the comment.
```suggestion
      for (j = 0; j < k; ++j) {
        sum += x_data[j * size + i] * alpha_data[j];
```